sdcm/cluster.py

```diff
@@ -3421,6 +3421,10 @@ def terminate_node(self, node: BaseNode, scylla_shards: int = 0) -> None:
             ))
         if node in self.nodes:
             self.nodes.remove(node)
+        ScyllaLogCollector([node],
```

Comment:
this would is one BaseCluster, so this would happen for all type of nodes...
we probably would want to limit it to scylla nodes only.

Reply:
on the other side of it, we never call `terminate_node` for loader and monitor nodes...
and I can't think of a case we might do that, but still I would try to limit it, and make it clearer what it's aimed for.

Reply:
here we specify just one node - and it's not collecting other db nodes logs because we don't provide `local_search_path` to `collect_logs` method